compiler/resolution/functionResolution.cpp

```diff
@@ -3514,24 +3534,29 @@ void resolveNormalCall(CallExpr* call) {
   if (call->partialTag && (!best || !best->fn ||
                            !best->fn->hasFlag(FLAG_NO_PARENS))) {
     if (best != NULL) {
-      delete best;
+      // best is deleted below with the other candidates
       best = NULL;
```

Comment:
This is a little worrisome.  If we set best to NULL, doesn't that prevent us from freeing its previous value?